modules/flowable-job-service/src/main/java/org/flowable/job/service/JobServiceConfiguration.java

```diff
@@ -524,9 +505,9 @@ public List<AsyncRunnableExecutionExceptionHandler> getAsyncRunnableExecutionExc
     }
 
     public JobServiceConfiguration setAsyncRunnableExecutionExceptionHandlers(List<AsyncRunnableExecutionExceptionHandler> asyncRunnableExecutionExceptionHandlers) {
-        this.asyncRunnableExecutionExceptionHandlers = asyncRunnableExecutionExceptionHandlers;
-        return this;
-    }
+      this.asyncRunnableExecutionExceptionHandlers = asyncRunnableExecutionExceptionHandlers;
+      return this;
+  }
```

Comment:
I think I missed this formatting change earlier. 

```suggestion
        this.asyncRunnableExecutionExceptionHandlers = asyncRunnableExecutionExceptionHandlers;
        return this;
    }
```

Reply:
these are really weird, I wonder what my Eclipse is doing, sorry for the hassle..

Reply:
all good. We have some code style in https://github.com/flowable/flowable-engine/tree/main/ide-settings. Separate for IntelliJ and Eclipse. However, I think that the one for Eclipse might not be up to date 😄